util/runpytest.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+# Should be run in the directory where the log files should be saved.
+# It should also have ../mic-odm-yaml/ directory, which contains all the microscopes files.
+# Ex:
+# cd odemis-testing/
+# ../odemis/util/runtests.sh 2>&1 | tee test-$(date +%Y%m%d).log
```

Comment:
shouldn't this now be runpytest.sh: `# ../odemis/util/runpytest.sh 2>&1 | tee test-$(date +%Y%m%d).log`?